h/static/scripts/tests/controllers/lozenge-controller-test.js

```diff
@@ -26,6 +26,40 @@ describe('LozengeController', function () {
     assert.equal(lozengeContentEl.textContent, opts.content);
   });
 
+  it('creates a new lozenge for a known named query term inside the container provided', function () {
+    var facetNameEl;
+    var facetValueEl;
+
+    el = document.createElement('div');
+    opts = {
+      content: 'user:foo',
+    };
+
+    new LozengeController(el, opts);
+    lozengeEl = el.querySelector('.js-lozenge');
+    lozengeContentEl = lozengeEl.querySelector('.js-lozenge__content');
```

Comment:
I would suggest extracting these `querySelector()` calls into helper functions. This way individual tests can be more concise and you don't have to update every single test if you change the UI structure of the component.